docs/guide/ingress/annotation.md

```diff
@@ -127,41 +127,44 @@ Traffic Routing can be controlled with following annotations:
 Access control for LoadBalancer can be controlled with following annotations:
 
 - <a name="scheme">`alb.ingress.kubernetes.io/scheme`</a> specifies whether your LoadBalancer will be internet facing. See [Load balancer scheme](http://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/how-elastic-load-balancing-works.html#load-balancer-scheme) in the AWS documentation for more details.
-    
+
     !!!example
         ```
         alb.ingress.kubernetes.io/scheme: internal
         ```
 
 - <a name="inbound-cidrs">`alb.ingress.kubernetes.io/inbound-cidrs`</a> specifies the CIDRs that are allowed to access LoadBalancer.
-    
+
     !!!warning ""
         this annotation will be ignored if `alb.ingress.kubernetes.io/security-groups` is specified.
-        
+
     !!!example
         ```
         alb.ingress.kubernetes.io/inbound-cidrs: 10.0.0.0/24
         ```
-            
+
 - <a name="security-groups">`alb.ingress.kubernetes.io/security-groups`</a> specifies the securityGroups you want to attach to LoadBalancer.
-    
+
+    !!!note ""
+        When this annotation is not present, the controller will automatically create 2 security groups: the first security group will allow access from `0.0.0.0/0` to the `listen-ports` and be attached to the ALB. The second security group will allow all TCP traffic from the first security group created for the ALB and be attached to the EC2 instances.
```

Comment:
we should change 0.0.0.0/0 to [`inbound-cidrs`](#inbound-cidrs) 😄 ,since it's configurable.
and [`listen-ports`](#listen-ports)

Reply:
overall lgtm to me with minimal comment 😄

Reply:
BTW, you are preview docs using python3 -m mkdocs serve to preview docs.
I'll add an pipenv file and make target like make docs-serve make docs-deploy to make this easiler

Reply:
Thanks for reviewing. Pushed a new commit with these changes.